packages/react/src/components/Chip/_ChipButton/ChipButton.module.css

```diff
@@ -0,0 +1,57 @@
+.chipButton {
+  --fdsc-chip-height: var(--fds-sizing-6);
+  --fdsc-chip-padding: var(--fds-spacing-2);
+  --fdsc-chip-background: var(--fds-semantic-surface-neutral-default);
+  --fdsc-chip-text-color: var(--fds-semantic-text-neutral-default);
+  --fdsc-chip-border: var(--fds-semantic-border-neutral-default);
+  --fdsc-chip-border-radius: var(--fds-border_radius-full);
+
+  background: var(--fdsc-chip-background);
+  padding: 0 var(--fdsc-chip-padding);
+  min-height: var(--fdsc-chip-height);
+  border-radius: var(--fdsc-chip-border-radius);
+  border: 1px solid var(--fdsc-chip-border);
+  color: var(--fdsc-chip-text-color);
+  display: block;
+  text-decoration: none;
+}
+
+.chipButton:hover {
+  --fdsc-chip-background: var(--fds-semantic-surface-neutral-subtle-hover);
+  --fdsc-chip-text-color: var(--fds-semantic-text-neutral-default);
+  --fds-chip-border: var(--fds-semantic-border-neutral-default);
+
+  cursor: pointer;
+}
+
+.chipButton:focus {
+  --fdsc-chip-background: var(--fds-semantic-surface-action-primary-active);
```

Comment:
Should the button be using the same color for focus as it does when it's active?

It looks kind of odd when I tab to it and it changes into an active color so I think it's active and then I tab away and it's no longer active.
Or in the case of a chip being selected and then I click on it to deselect it, but it remains looking like it's selected afterwards. I would have to click somewhere else to see it's true state and that would be really confusing I'd imagine 😅

Reply:
Great catch! It was wrong focus style. It's fixed now. :)